CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Reference distribution changes from RHEL/CentOS 6 to RHEL/CentOS 7

```

Comment:
changed?

Reply:
I think we tend to use present tense in CNR...

NTC